mattermost-plugin/webapp/src/index.tsx

```diff
@@ -319,6 +319,21 @@ export default class Plugin {
                 ),
                 false
             )
+
+            // Insights handler
+            if (this.registry?.registerInsightsHandler) {
+                this.registry?.registerInsightsHandler(async (timeRange: string, page: number, perPage: number, teamId: string, insightType: string) => {
+                    if (insightType === 'MY') {
```

Comment:
nit: Probably a constant storing the types somewhere can be more explicit an clear.

Reply:
cc: @BenCookie95 could you please make this small change in [this](https://github.com/mattermost/focalboard/pull/3596) PR?

Reply:
The branch is in upstream focalboard for convenience.

Reply:
Done!

Reply:
https://github.com/mattermost/focalboard/pull/3596/commits/9e8cab2b83cc30df826b87d2e40a6aa2e7230934